src/iam/certmodules/pkcs11/pkcs11.cpp

```diff
@@ -667,7 +667,7 @@ Error PKCS11Module::FindObject(pkcs11::SessionContext& session, const SearchObje
         templ.EmplaceBack(CKA_LABEL, Array<uint8_t>(labelPtr, filter.mLabel.Size()));
     }
 
-    if (!filter.mType.HasValue()) {
+    if (filter.mType.HasValue()) {
```

Comment:
Commit message:  `didn't consider input class` what does it mean?

Reply:
mType value  specifies pkcs11 class like CKO_CERTIFICATE, CKO_PUB_KEY. this value was ignored due to logical error